azkaban-web-server/src/main/java/azkaban/webapp/servlet/ScheduleServlet.java

```diff
@@ -90,8 +90,8 @@ private void handleAJAXAction(final HttpServletRequest req,
       ajaxSlaInfo(req, ret, session.getUser());
     } else if (ajaxName.equals("setSla")) {
       ajaxSetSla(req, ret, session.getUser());
-    } else if (ajaxName.equals("loadFlow")) {
```

Comment:
The change itself makes sense, but my concern is it's not backward compatible.

Reply:
Yes. I suggested making it backward compatible but understood from @HappyRay's comment that he'd prefer a "blunt rename" :) ..it would be as simple as this:

```java
// alias loadFlow is preserved for backward compatibility
} else if (ajaxName.equals("fetchSchedules") || ajaxName.equals("loadFlow")) {
```

Should I change to this?

Reply:
+1 on this.

Reply:
Oh, this comment is "on the wrong side" so it won't go away :) But i have updated it, see the diff, please.